### PR TITLE
Ratest-281: Fixes the naming of the patient condition pill

### PR DESCRIPTION
**Pull request by Irenyak1**
Work is part of  [RATEST-281](https://issues.openmrs.org/browse/RATEST-281)

Fixes -> Naming of the patient condition pill in the readme file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Install & Upgrade Tests |
 [All Chrome Tests](https://ci.openmrs.org/browse/CONTRIB-QA) ![Build Status](https://ci.openmrs.org/plugins/servlet/wittified/build-status/CONTRIB-QA)
 [![All Firefox Tests](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/qa.yml/badge.svg?branch=master)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/qa.yml)
 [![RefApp 2.x Clinical Visit](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-clinical-visit.yml/badge.svg)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-clinical-visit.yml)
-[![RefApp 2.x Patient Condition](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2.x-condition.yml/badge.svg)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2.x-condition.yml)
```
Comment (sherrif10) [inline README.md:27]: Fixed it in [here](https://github.com/openmrs/openmrs-contrib-qaframework/pull/276/files)

Comment (Irenyak1) [reply README.md:27]: cc @kdaud

**Review by sherrif10 — APPROVED** [review]
Thanks

**Comment by Irenyak1** [comment]
@sherrif10  and @kdaud  could this be merged into master?

**Review by kdaud — APPROVED** [review]

**Comment by kdaud** [comment]
@Irenyak1 have you taken a look at [RATEST-874](https://issues.openmrs.org/browse/RESTWS-874)?

**Pull request merged**